telegram/files/file.py

```diff
@@ -111,33 +111,39 @@ def download(
               the path of a local file (which is the case when a Bot API Server is running in
               local mode), this method will just return the path.
 
+        .. versionchanged:: 14.0
+            * :param:`custom_path` now also accepts pathlib.Path objects as argument.
```

Comment:
```suggestion
            * ``custom_path`` now also accepts pathlib.Path objects as argument.
```

the `:param:` directly doesn't render correctly here

Reply:
done